pkg/kubelet/kubelet_pods_test.go

```diff
@@ -2920,8 +2920,8 @@ func Test_generateAPIPodStatus(t *testing.T) {
 					ready(waitingWithLastTerminationUnknown("containerB", 0)),
 				},
 			},
-			expectedPodHasNetworkCondition: v1.PodCondition{
-				Type:   kubetypes.PodHasNetwork,
+			expectedPodReadyToStartContainersCondition: v1.PodCondition{
+				Type:   kubetypes.PodReadyToStartContainers,
```

Comment:
Just to be 100% certain, we are renaming an alpha condition name that was introduced in the same KEP?  Ie PodHasNetwork has never been beta or GA?

Reply:
@ddebroy I am a bit confused on this.  https://github.com/kubernetes/enhancements/blob/master/keps/sig-node/3085-pod-conditions-for-starting-completition-of-sandbox-creation/kep.yaml updates it says it was moved to beta but I think we talked about renaming and keeping it in alpha for now. 

ref: https://github.com/kubernetes/kubernetes/pull/117702#pullrequestreview-1408251857

I think the KEP was updated but the code was not moved to beta.

Reply:
I just want to make completely certain we're not renaming a condition that is in beta / on-by-default, or not renaming an existing condition that is GA to a new condition (as part of an alpha feature).  That's a more involved process than just a rename for api compatibility reasons.

Reply:
So in the feature gate code it says this:

```
	PodHasNetworkCondition: {Default: false, PreRelease: featuregate.Alpha},
```

That should mean it is still in Alpha stage?

Reply:
Ok, thanks.

I will probably have comments on how this condition is reported up as I’m actively trying to unify and optimize status reporting (which can have significant latency due to various factors).  Reading the KEP i think the initial approach is fine but we need to clarify the rules about how this is interpreted by latency sensitive clients.  I specifically want to preserve the kubelets ability to aggregate status writes during load, and prioritize certain writes over others (ie a pod transitioning to ready or unready is more important than other transitions). #116617 tracks that.